### ISSUE TITLE
Windows 7, Open Atom And Crash

### ISSUE THREAD

**Issue by Chandralim** [atom/atom#11970]
![atom crash](https://cloud.githubusercontent.com/assets/19898897/15996897/5c28f02e-3157-11e6-8b80-238114966fcb.jpg)

the atom just crashed when i open atom.exe

please help me !!!

**Comment by Ben3eeE** [comment]
Please fill out the provided issue template. As it is now it is not possible to do anything about this because it lacks information.

Information you can include and things to test:
- Does Atom start normally in safe mode? You can enter safe mode by running `atom --safe`.
- Does running `atom --clear-window-state` make the problem disappear?
- When did the problem start?
- What is your current version of Atom?
- Upload the latest crash dump file. It is located in `%LocalAppData%\Temp\Atom Crashes`

**Comment by Chandralim** [comment]
Does Atom start normally in safe mode? You can enter safe mode by running atom --safe.
( No, Still Crash )
Does running atom --clear-window-state make the problem disappear?
( No, Still Crash too )
When did the problem start?
( when open the atom application or when execute it )
What is your current version of Atom?
( the atom version is 1.8 )
Upload the latest crash dump file. It is located in %LocalAppData%\Temp\Atom Crashes
(
 _companyName:GitHub
_productName:Atom
_version:1.8.0
platform:win32
process_type:renderer
prod:Electron
ptime:5567
rept:electron-crash-service
ver:0.36.8
)

can you help me please... ^-^

**Comment by notJackson** [comment]
Same here! Started today, dunno if it silent updated.

>Does Atom start normally in safe mode? You can enter safe mode by running atom --safe.

Crashes.
>Does running atom --clear-window-state make the problem disappear?

Crashes.
>When did the problem start?

13.06.2016 (dd-mm-yyyy)
>What is your current version of Atom?

1.8 (No idea when it silence-updated the last time.)
>Upload the latest crash dump file. It is located in %LocalAppData%\Temp\Atom Crashes

http://www.file-upload.net/download-11674485/AtomCrashes.rar.html

>Did you try deleting all visible .atom / atom / programfiles/atom files and a fresh install?

Still crashing.

It works when I start it (crashes! -> keep open -> Open Project Folder)
Unless I visit my current work DIR which I want to continue working on.

**Comment by Chandralim** [comment]
i have get the solution. works for me... just deleting the ~/.atom/blob-store . then all work find... 
thanks for @ijgithub, and  @50Wliu for the solution... that help me so much thank you.... 
the link https://github.com/atom/atom/issues/10397

**Comment by notJackson** [comment]
[Cause of my problem folder named "name."](https://github.com/atom/atom/issues/10397#issuecomment-214555730)

**Comment by JoeyFerg** [comment]
I have the same issue. I deleted the blob-store folder and it still crashes. Then I check back in the folder and the blob-store folder recreated itself. I do not have a folder named "name." like some of you guys.

I have also tried `atom --safe` and `atom --clear-window-state` and neither of them work.

If anyone has a fix for this (other than deleting the blob-store folder) I would greatly appreciate it.
